src/codegen/riscv64/constants-riscv64.h

```diff
@@ -724,6 +783,8 @@ class InstructionBase {
     kCAType,
     kCBType,
     kCJType,
+    // V extension
+    kVtype,
```

Comment:
Perhaps we should refer to the naming of RVC extensions according to page 100  of riscv-spec-20191213.pdf. 
How about naming the v-types according to the major opcode? i.e., LOAD-FP/STORE-FP/AMO/OPV.